src/prometheus_cowboy.erl

```diff
@@ -41,13 +41,13 @@ init([]) ->
 ?DOC(false).
 -spec to_cowboy_headers([{atom(), binary()}]) -> [{binary(), binary()}].
 to_cowboy_headers(RespHeaders) ->
-    lists:map(fun to_cowboy_headers_/1, RespHeaders).
+    lists:map(fun to_cowboy_headers_1/1, RespHeaders).
```

Comment:
do you think that "to_cowboy_headers_1" is better than "to_cowboy_headers_"?

Reply:
isn't it `to_cowboy_header`?

Reply:
elvis complains about atoms and function names ending in `_/` and I don't really mind just leaving elvis with its defaults :)